src/sections/app.style.js

```diff
@@ -3,7 +3,7 @@ import { createGlobalStyle } from "styled-components";
 const GlobalStyle = createGlobalStyle`
 
 ::selection {
-    background: rgba(0, 70, 0, 0.99);
+    background: rgb(0, 179, 159);
```

Comment:
Are we sure to use hard coded values? Or shall we use from theme colors?

Reply:
the color code was provided by @Chadha93

Reply:
But he also mentioned from "theme" in his issue ticket, please check and confirm. 
// @Chadha93

Reply:
@samyakshah3008 can you guide me where to find the theme